sigstore-gradle/sigstore-gradle-sign-base-plugin/src/main/kotlin/dev/sigstore/sign/work/SignWorkAction.kt

```diff
@@ -65,10 +65,7 @@ abstract class SignWorkAction : WorkAction<SignWorkParameters> {
                 payloadHash = HashValue(
                     // See https://github.com/sigstore/sigstore-java/issues/85
                     algorithm = HashAlgorithm.sha256,
-                    // https://github.com/sigstore/sigstore-java/issues/86
-                    hash = result.digest.chunked(2)
-                        .map { it.toInt(16).toByte() }
-                        .toByteArray(),
+                    hash = result.digest,
```

Comment:
This would create merge conflict with bundle pr, so I would rather merge bundle the first

Reply:
Sure. Can wait on both